### DISCUSSION TITLE
Building and exporting ionic-cordova app with quasar command throw an exit code 1, even though app is built and exported successfully

### DISCUSSION THREAD

**Discussion by icarusdust** [quasarframework/quasar#14368] [CLI - Capacitor / Cordova modes]
Building with ionic `cordova build ios --release --device `works as expected but when I am turning to quasar command  `quasar build -m ios -- --release --device` it works with an unexpected error:

```
** ARCHIVE SUCCEEDED **
2022-09-12 11:08:11.075 xcodebuild[2595:20171] [MT] IDEDistribution: -[IDEDistributionLogging _createLoggingBundleAtPath:]: Created bundle at path "/var/folders/m7/h1mg7c7x40ddjz6mxjxm3htr0000gn/T/Quasar App_2022-09-12_11-08-11.073.xcdistributionlogs".
Exported Quasar App to: /Users/builder/clone/quasar-project/src-cordova/platforms/ios/build/device
** EXPORT SUCCEEDED **

Build failed :|
Step 7 script `Build iOS` exited with status code 1
```

I can generate .ipa regardless of this error. I wonder if you could point me to the right direction about what this error is and how to solve it ?

**Comment by eranshmil** [comment]
I also had this issue when using Quasar v1 with node.js v16. Downgrading to node.js v14 solved this issue for me.
